jdhzzz-ct-usa.md

```diff
@@ -0,0 +1,5 @@
+# CT Near Hartford
+
+## Places to get beer
+- Corner Pug in West Hartford
+- Eli Cannon's in Middletown
```

Comment:
Ha.  I noticed that atom does that automatically.

Did he say why?  (I have an answer, but want to know his)